packages/big-design/src/components/StatusMessage/spec.tsx

```diff
@@ -0,0 +1,92 @@
+import { theme } from '@bigcommerce/big-design-theme';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import 'jest-styled-components';
+
+import { Button } from '../Button';
+
+import { StatusMessage } from './index';
+
+const variants = [
+  'info',
+  'error',
+  'success',
+  'warning',
+  'unauthorized',
+  'search',
+  'server-error',
+  '404',
+] as const;
+
+test('renders basic status message', () => {
+  render(<StatusMessage message="Basic status message" />);
+
+  expect(screen.getByText('Basic status message')).toBeInTheDocument();
+});
+
+test('renders status message illustration', () => {
+  render(<StatusMessage message="Basic status message" />);
+
+  expect(screen.getByRole('figure', { hidden: true })).toBeInTheDocument();
+});
+
+test('renders panel status message', () => {
+  render(<StatusMessage heading="Panel heading" message="Basic status message" />);
+
+  expect(screen.getByRole('heading', { level: 4, name: 'Panel heading' })).toBeInTheDocument();
+});
+
+test('renders page status message', () => {
```

Comment:
as above

also, if there difference is the expected heading level, we can call that more clearly in the test description,
we can also use a describe block to make the `page` size context really explicit, e.g.

```tsx
describe('when at "page" size', () => {
  test('renders the heading as a H1', () => {
    render(<StatusMessage heading="Page heading" message="Basic status message" size="page" />);

    expect(screen.getByRole('heading', { level: 1, name: 'Page heading' })).toBeInTheDocument();
  });
});
```